libs/utils/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./lib/assertNotEmpty";
+export * from "./lib/ensureNotEmpty";
```

Comment:
Those two functions are not yet part of corelibrary, but they should and IMO we should use corelib versions instead of copying it to contracts generator repo